src/components/CoreLabel/index.tsx

```diff
@@ -1,22 +1,65 @@
 import CoreLogo from 'components/CoreLogo';
-import {SFC} from 'types';
+import {AppDispatch, Core, SFC} from 'types';
 import * as S from './Styles';
+import Line from 'components/Line';
+import {useToggle} from 'hooks';
+import {useDispatch, useSelector} from 'react-redux';
+import {getSelf} from 'selectors/state';
+import {deleteCore} from 'dispatchers/cores';
+import DropdownMenu from 'components/DropdownMenu';
+import CoreModal from 'modals/CoreModal';
+import {mdiDeleteOutline, mdiDotsVertical, mdiSquareEditOutline} from '@mdi/js';
 
 export interface CoreLabelProps {
   domain: string;
   logo: string | null;
   ticker: string;
+  core: Core;
 }
 
-const CoreLabel: SFC<CoreLabelProps> = ({className, domain, logo, ticker}) => {
+const CoreLabel: SFC<CoreLabelProps> = ({className, domain, logo, ticker, core}) => {
```

Comment:
If we are passing the core object to this component, then I'd suggest we don't pass the order fields: `domain`, `logo`, `ticker`.
Since they should already be there in the core itself.